code/__DEFINES/particles.dm

```diff
@@ -3,3 +3,10 @@
 
 /// If we're inside something inside a mob, display off that mob too
 #define PARTICLE_ATTACH_MOB (1<<0)
+
+#define DEBRIS_SPARKS "spark"
```

Comment:
Please add a comment above this line saying these defines are used in explosions for their icon state.